Mathlib/Geometry/Manifold/Algebra/LieGroup.lean

```diff
@@ -65,7 +70,10 @@ class LieGroup {𝕜 : Type*} [NontriviallyNormedField 𝕜] {H : Type*} [Topolo
   smooth_inv : Smooth I I fun a : G => a⁻¹
 #align lie_group LieGroup
 
-section LieGroup
+/-! Let `f : M → G` be `C^n` or smooth functions into a smooth Lie group, then `f` is point-wise
+  invertible with smooth inverse `f`. If `f` and `g` are two such functions, the quotient
+  `f / g` (i.e., the point-wise product of `f` and the point-wise inverse of `g` is also smooth. -/
```

Comment:
Can you add a section header, like so? I assume `to_additive` does some of this? IF not, remove the references to negation and subtraction.
```suggestion
/-!
### Smoothness of inversion, negation, division and subtraction

Let `f : M → G` be `C^n` or smooth functions into a smooth Lie group, then `f` is point-wise
  invertible with smooth inverse `f`. If `f` and `g` are two such functions, the quotient
  `f / g` (i.e., the point-wise product of `f` and the point-wise inverse of `g` is also smooth. -/
```

Reply:
Indeed, to_additive generates corresponding lemmas for negation and subtraction.